src/client/voice/receiver/PacketHandler.js

```diff
@@ -78,6 +79,23 @@ class PacketHandler extends EventEmitter {
     const ssrc = buffer.readUInt32BE(8);
     const user = this.userFromSSRC(ssrc);
     if (!user) return;
+
+    if (this.speakingTimeouts.get(ssrc)) {
```

Comment:
```suggestion
    if (this.speakingTimeouts.has(ssrc)) {
```

`has` should be used for consistency